### PR TITLE
Fix Memory Leak

### PR DESCRIPTION
**Pull request by anynon**

### CODE REVIEW

**Review by niwenxin — APPROVED** [review]

**Comment by 3h6-1** [comment]
You forgot a semicolon after it, so this won't compile.

**Comment by ren7995** [comment]
lol

File: cicuta_virosa.c [hunk 1]
```diff
@@ -238,6 +238,7 @@ void cicuta_virosa(void)
     uint64_t id = 0;
     redeem_racers = calloc(1, REDEEM_RACERS_COUNT * sizeof(pthread_t));
     increase_limits(10240);
+    free(race_flag)
```
Comment (tylerwolf35) [inline cicuta_virosa.c:241]: Needs a semicolon to compile.

**Comment by Hochgesand** [comment]
lol

**Comment by flopana** [comment]
lol

**Comment by brandonplank** [comment]
Don't do this.....

**Comment by pwned4ever** [comment]
> Don't do this.....

Agreed, it breaks the exploit and doesn't do what said user pretends it does. Memory still leaking?

**Pull request closed without merging**